CHANGELOG.rst

```diff
@@ -70,6 +70,9 @@ time or change existing behaviors in order to make them less surprising/more use
   namespace in which doctests run.
   Thanks `@milliams`_ for the complete PR (`#1428`_).
 
+* New ``--doctest-report`` option available to change the output format of diffs
```

Comment:
Nit pick: "when displaying failed doctests (#1749). Thanks `@hartym` for the PR." :wink: